### DISCUSSION TITLE
Error: admission webhook "vscaledobject.kb.io" denied the request

### DISCUSSION THREAD

**Discussion by ayushk048-Tandem** [kedacore/keda#5220] [Q&A / Need Help] [answered]
```
Error: admission webhook "vscaledobject.kb.io" denied the request: the workload 'cloud-svc' of type 'apps/v1.Deployment' is already managed by the hpa 'cloud-svc'.
```
Helm deployment failed with the above error. we are using HPA and Keda config at the same time

**Comment by JorTurFer** [comment, marked as answer]
> Helm deployment failed with the above error. we are using HPA and Keda config at the same time

You cannot use HPA and ScaledObject at the same time (nor 2 HPA or 2 ScaledObjects) for the same workload. You have to choose or HPA or ScaledObject. This is because ScaledObject generates an HPA under the hood and the HPA controller treats each HPA as individual item, so if you have your HPA and you ScaledObject scaling the same workload, the HPA controller can scale out and scale in the workload all the time because each HPA requires different replicas
